app/ethos/management/commands/distribute_ethos.py

```diff
@@ -0,0 +1,69 @@
+
+            # Instantiate EthOS contract
+            contract = w3.eth.contract(settings.ETHOS_CONTRACT_ADDRESS, abi=abi)
+
+            tx = contract.functions.transfer(address, n * 10**18).buildTransaction({
+                'nonce': nonce + i,
+                'gas': 100000,
+                'gasPrice': recommend_min_gas_price_to_confirm_in_time(5) * 10**9
```

Comment:
Can we move `recommend_min_gas_price_to_confirm_in_time(5) * 10**9` out of this for loop and define it once and reference that var here?